Svc/Health/test/int/test_cmd_health.py

```diff
@@ -0,0 +1,153 @@
+""" test_cmd_health.py:
+
+Test the command dispatcher with basic integration tests.
+"""
+import time
+from fprime_gds.common.testing_fw import predicates
+from fprime_gds.common.utils.event_severity import EventSeverity
+
+def test_send_health_command(fprime_test_api):
+    """Test that commands may be sent
+
+    Tests command send, dispatch, and receipt using send_and_assert command with a pair of health commands.
+
+     health.HLTH_ENABLE, [disabled/enabled}
+     health.HLTH_PING_ENABLE,["Ref_pingRcvr","DISABLED/ENABLED"]
+     health.HLTH_CHNG_PING,["Ref_pingRcvr",1,1]
+    """
+
+    cmd_events = fprime_test_api.get_event_pred(severity=EventSeverity.COMMAND)
+    actHi_events = fprime_test_api.get_event_pred(severity=EventSeverity.ACTIVITY_HI)
+    warnLo_events = fprime_test_api.get_event_pred(severity=EventSeverity.WARNING_LO)        
+
+    pred = predicates.greater_than(0)
+    zero = predicates.equal_to(0)
+    
+    ## Read NumPings
+    pingRcvr = fprime_test_api.await_telemetry(fprime_test_api.get_mnemonic('Ref.PingReceiver') + "." + "PR_NumPings", start="NOW")
+    # If no constraints are specified on the channels, the predicate will always return true    
+    ping_change = fprime_test_api.get_telemetry_pred(fprime_test_api.get_mnemonic('Ref.PingReceiver') + "." + "PR_NumPings", pingRcvr)
+    
+    # Expect number still increment after clear_history
+    fprime_test_api.clear_histories()  # will clear all history (can read telemetry channel again with latest value.  otherwise still have old value)
+    time.sleep(5)
+    
+    pingRcvra = fprime_test_api.await_telemetry(fprime_test_api.get_mnemonic('Ref.PingReceiver') + "." + "PR_NumPings", start="NOW")
+    # If no constraints are specified on the channels, the predicate will always return true    
+    pinga_change = fprime_test_api.get_telemetry_pred(fprime_test_api.get_mnemonic('Ref.PingReceiver') + "." + "PR_NumPings", pingRcvra)
+     
+    # Disable the health  HLTH_ENABLE command         
+    fprime_test_api.send_and_assert_command(fprime_test_api.get_mnemonic('Svc.Health') + '.' + 'HLTH_ENABLE',["DISABLED"])
+    assert fprime_test_api.get_command_test_history().size() == 1   # current command count    
+
+    fprime_test_api.assert_event_count(pred, cmd_events)
+    fprime_test_api.assert_event_count(pred, actHi_events)
+
+    # Expect number no change (stale or stop)    
+    pingRcvr1 = fprime_test_api.await_telemetry(fprime_test_api.get_mnemonic('Ref.PingReceiver') + "." + "PR_NumPings", start="NOW")
```

Comment:
Another usage of a `Ref` component. You might want to move all these tests of Ref components to fprime's Ref/test/int directory.